### ISSUE TITLE
Word limit - JOSS review

### ISSUE THREAD

**Issue by mrocamora** [tuomaseerola/onsetsync#3]
@tuomaseerola the word count for the paper.md file is 4834 (according to the editorial bot), surpassing the word limit. 
Editor (@faroit) suggests that even if most of the word count is driven by the tables, the paper is too long and should be shorten by at least 1-2 pages.

**Comment by tuomaseerola** [comment]
Of course, this silly of us not following the upper limit. We have now eliminated half of the content from the paper (it is now 2508 words) by taking several sections and multiple graphs as additional documentation articles: 

- [Advanced topics](https://tuomaseerola.github.io/onsetsync/articles/advanced_topics.html) (which contains 3 sub-topics from the paper)
- [Analysis of periodicity](https://tuomaseerola.github.io/onsetsync/articles/analysisis_of_periodicity.html)

We also streamlined the text to be shorter.
